src/governance.rst

```diff
@@ -0,0 +1,173 @@
+* **maintainers:** A maintainer is an individual responsible for the management
+  of one or more feedstock repositories and packages. Maintainers have the ability
+  to merge pull requests into the feedstock of only the packages they maintain.
+* **external contributors:** This group encompasses all others who are not on
+  core, part of staged-recipes, or maintainers. This includes first time
+  contributors, collaborators, and funders. They have no special rights within
+  the conda-forge organization itself.
+
+Sub-Teams
+---------
+The core team may elect to create new sub-teams for managing the daily business
+of the organization. While sub-teams may have non-core members, every sub-team
+must have at least one core team member at all times. If a sub-team fails to
+have  a core team member for more than 1 week, that team is considered to be
+dissolved. A new sub-team would need to be established by the core in order to
+reinstate the activity.
+
+Sub-teams have a charter that is either *dynamic* or *static*.
+
+* A *dynamic* charter means that the sub-team is self-organizing, with respect
+  to its own internal policies, procedures, and membership. A sub-team may choose
+  to modify its membership independent of the core team. For example, a
+  Google Summer of Code team could be a good candidate for a dynamic charter.
+  Alternatively, language-based maintenance teams also have a dynamic charter.
+* A *static* charter means that all membership decisions and non-trivial policies
+  changes must be approved by the core team. For example, a finance team
+  may require a static charter.
+
+All sub-teams must adhere to the governance, policies, and procedures of
+conda-forge at all times.
+
+Voting
+------
+This section presents descriptions and criteria for voting items in the
+conda-forge community. The core team is the only team with voting rights.
+Members of core may also call a vote on any topic. The restrictions on
+calling a vote are as follows:
+
+* There must only be one vote active on a particular item at any time.
+* The act of calling for a vote cannot itself violate the code of
+  conduct. For example, Sam repeatedly called for votes immediately
+  after a previous votes failed to achieve Sam's result. Sam is
+  attempting to bully other members of core into agreeing, and is thus
+  violating the code of conduct.
+
+Voting items are labeled as either **standard** or **sensitive**.
+Standard items are ones where public record and discourse and record is
+preferable. Sensitive voting items are ones where the results of the
+vote should remain private to the voters after the vote has occurred.
+Sensitive votes should take place on
+`the Helios voting system <https://vote.heliosvoting.org/>`_ in order
+retain anonymity.
+
+The default voting period is 1 week (7 days). This may be modified at
+the time when a vote is called, but may never be less than 24 hrs.
+
```

Comment:
In the following the term "majority" is used -- is that a majority of the entire core team? or a majority of people that placed a vote on the particular issue?

If the former, then it may be very hard to attain a majority quickly when needed. If the later, than a minimum quorum should be defined.

In any case, we need a bit of clarifying language.